frontend/src/components/File.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-  <tr v-on:click="selectRow">
+  <tr @click="selectRow">
     <td class="text-left">
       <div class="custom-control custom-checkbox">
-        <input v-model="isSelected" type="checkbox" class="form-check-input"/>
+        <input :checked="isSelected" @change="selectFile" type="checkbox" class="form-check-input selection-check-box"/>
```

Comment:
`v-model` had to be removed because we needed something else than the default syntatic sugar provided by `v-model` this checkbox needs to be in sync with the file state.